clang/test/Analysis/nullability.c

```diff
@@ -51,3 +54,15 @@ int *cannot_return_null() {
 __attribute__((returns_nonnull)) int *passthrough(int *p) {
   return p; // no-warning: we have no evidence that `p` is null, i.e., violating the contract
 }
+
+__attribute__((noreturn))
+void exit(int);
```

Comment:
Is this declaration used somehow?

Reply:
It is not used, indeed; thanks for spotting it!
It was relevant in one of the working versions of the test case, but not in the committed one. Removed in 41956123